packages/arb-util/broadcaster/clientmanager.go

```diff
@@ -238,7 +238,9 @@ func (cm *ClientManager) verifyClients() {
 	var deadClientCount uint64
 	var aliveClientCount uint64
 	for client := range cm.clientPtrMap {
+		client.timeoutMutex.Lock()
```

Comment:
I think it'd be a little cleaner to keep the mutex stuff in `ClientConnection` and just add a public `GetLestHeard()` that does the mutex control

Reply:
:+1: